_includes/api_reference.html

```diff
@@ -0,0 +1,28 @@
+<div id="doccontent">
+        <div class="doccontent-api-reference">
```

Comment:
This _includes file might not be needed (you can revert back to using document.html). I dont think you need to inject a new div class here and should be able to override the existing div class "sect1"? The "sect1" class is already in the both of the API .html files plus we can avoid have duplicate javascript in multiple files.